build.gradle

```diff
@@ -1,13 +1,14 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 
 buildscript {
+    ext.kotlin_version = '1.2.71'
```

Comment:
we could bump this up to the latest kotlin version `1.3.10`

Reply:
Fixed